NetKAN/xScienceContinued.netkan

```diff
@@ -0,0 +1,20 @@
+{
+    "spec_version": 1,
+    "identifier": "xScienceContinued",
+    "name": "[x] Science! Continued",
+    "$kref": "#/ckan/github/Flupster/KSP-X-Science",
+    "license": "CC-BY-NC-SA-4.0",
+    "author": ["Z-Key Aerospace", "Brodrick", "Flupster"],
+    "abstract": "The Science Report and Checklist for KSP.  [x] Science! keeps track of the science experiments you have completed and recovered to Kerbal Space Center. It also tracks science you have gathered which is held on vehicles and kerbals.  No matter what game scene you are in, what you current vehicle is, [x] Science! always reports on all science held anywhere.",
+    "resources": {
+        "homepage": "http://forum.kerbalspaceprogram.com/index.php?/topic/107661-ksp-105-x-science",
```

Comment:
Suggestion: Wildcard URL.
Better suggestion: @Flupster Make a new forum thread for your continuation and link it instead.
```suggestion
        "homepage": "http://forum.kerbalspaceprogram.com/index.php?/topic/107661-*",
```